src/category_theory/limits/shapes/kernel_pair.lean

```diff
@@ -128,16 +124,16 @@ def comp_of_mono {f₁ : X ⟶ Y} {f₂ : Y ⟶ Z} [mono f₂] (small_k : is_ker
     intros m m₁ m₂,
     apply small_k.is_limit.hom_ext,
     refine ((pullback_cone.mk a b _) : pullback_cone f₁ _).equalizer_ext _ _,
-    rwa (pullback_cone.is_limit.lift' small_k.is_limit s.fst s.snd _).2.1,
-    rwa (pullback_cone.is_limit.lift' small_k.is_limit s.fst s.snd _).2.2,
-  end }
+    { erw (pullback_cone.is_limit.lift' small_k.is_limit s.fst s.snd _).2.1, assumption },
```

Comment:
I'm always hesitant about `erw` - is there a reason this is needed, and is there API we can instead add to make it not needed?